src/components/inspectors/InspectorPanel.vue

```diff
@@ -23,7 +23,6 @@ import {
   FormTextArea,
   FormCheckbox,
   FormRadioButtonGroup,
-  FormCodeEditor,
```

Comment:
Vue complained about this component not being found; it appears it's not actually used or defined anywhere.